### ISSUE TITLE
Problems encountered during the training of the model

### ISSUE THREAD

**Issue by ChloeTT-LAO** [garvita-tiwari/PoseNDF#32]
Dear PoseNDF team, 
Recently I encountered some problems when I run trainer.py on AMASS dataset. I found that no matter how I adjust the parameters, the loss function kept the same value unchanged after thousands of epochs having been iterated. I have no idea in which step I made mistakes, because I ran the code and generated the training dataset exactly the way you did. 
What's more, I 'm a little confused about these two data files in load_data.py: data_path and amass_dir. What is the difference between them? According to my understanding, data_path refer to training dataset generated by prepare_traindata.py and amass_dir refer to sampled AMASS dataset, is that right? And what do these two variableds, data_files and amass_files refer to?
Look forward to your reply!
Thanks in advance!
